waflyctl.go

```diff
@@ -2017,7 +2023,8 @@ var (
 	action           = app.Flag("action", "Action to take on the rules list and rule tags. Overwrites action defined in config file. One of: disabled, block, log.").Enum("disabled", "block", "log")
 	apiEndpoint      = app.Flag("apiendpoint", "Fastly API endpoint to use.").Default("https://api.fastly.com").String()
 	apiKey           = app.Flag("apikey", "API Key to use. Required.").Required().String()
-	backup           = app.Flag("backup", "Store a copy of the WAF configuration in "+homeDir()+"/.waflyctl-<service-id>.rules.").Bool()
+	backup           = app.Flag("backup", "Store a copy of the WAF configuration locally.").Bool()
+	backupPath       = app.Flag("backup-path", "Location for the WAF configuration backup file.").Default(homeDir() + "/waflyctl-backup-<service-id>.toml").String()
```

Comment:
Identified issue with default filename not being tokenised with SID. Do not merge